python/ray/train/tests/test_tune.py

```diff
@@ -299,6 +299,40 @@ def train_func():
     assert len(trial_dfs[0]["training_iteration"]) == 4
 
 
+def test_tune_torch_get_device(num_workers=1, num_gpus_per_worker=1):
```

Comment:
test_tune is not run in GPU test suite. Do we still need this test or is it a duplicate of the other tests that we have?

Reply:
i followed the previous style and the actual test is happening in `test_gpu.py`